spec/person_rspec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './spec_helper'
+
+describe Person do
+  it 'creates a new instance of the Person class' do
+    person = Person.new('age', 'name')
+    expect(person).to be_an_instance_of Person
+  end
+
+  it '#correct_name returns the name of the person object' do
+    name = Person.new(23, 'John')
+    expect(name.correct_name).to eq('John')
+  end
```

Comment:
- [ ] Good job with test coverage so far, but as we are aiming for 100% test coverage, it will be nice to cover all our methods in all our classes with tests, in the `Person` class  for example, the following methods should be covered with test
  - adding a new rentals
  - checking permissions
  - validating name
Please as it is a [requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md)for this project and best practice, please ensure all the methods in all the classes are covered with tests.